src/app/migrations/0002_licenserequest.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2018-07-01 09:23
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='LicenseRequest',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('fullname', models.CharField(max_length=70)),
+                ('shortIdentifier', models.CharField(max_length=25)),
+                ('submissionDatetime', models.DateTimeField(auto_now_add=True)),
+                ('userEmail', models.EmailField(max_length=35)),
+                ('xml', models.TextField()),
+            ],
+            options={
+                'verbose_name': 'LicenseRequest',
+                'verbose_name_plural': 'LicenseRequests',
+            },
+        ),
+    ]
```

Comment:
Again, this can be combined in the 001_initial.py migrations. Just delete your migration files and run `python manage.py makemigrations` and commit that file.

Reply:
After that you would have to run migrate command also. ( and createsuperuser too, if you want admin access)